src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,6 +203,18 @@ private void configureSubsystems() {
             tagVision.setCameraConsumer(
                     (m) -> drive.addVisionMeasurement(m.pose(), m.timestamp(), m.variance()));
         }
+        if (Constants.currentMode == Mode.REAL) {
+            scoringSubsystem = new ScoringSubsystem(new ShooterIOTalonFX(), new AimerIORoboRio());
+            intakeSubsystem = new IntakeSubsystem(new IntakeNEOVortex());
+        } else if (Constants.currentMode == Mode.SIM) {
+            scoringSubsystem = new ScoringSubsystem(new ShooterIOSim(), new AimerIOSim());
+            intakeSubsystem = new IntakeSubsystem(new IntakeIOSim());
+        }
+    }
+
+    public void enabledInit() {
+        intakeSubsystem.run(IntakeAction.NONE);
+        scoringSubsystem.setAction(ScoringAction.INTAKE);
```

Comment:
On enabled, the scoring subsystem probably shouldn't be attempting any specific action (would expect this to be idle)